_posts/2018-07-30-ARIA-states.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: ARIA States
+description: A quick introduction to ARIA's state system.
+author: Alex Brenon
+categories: basics
```

Comment:
should reclassify as background or quick tips.
@ericwbailey you got thoughts on where to put this?